packages/edit-site/src/components/save-button/index.js

```diff
@@ -47,13 +60,13 @@ export default function SaveButton( {
 	const getLabel = () => {
 		if ( isPreviewingTheme() ) {
 			if ( isSaving ) {
-				return __( 'Activating' );
+				return sprintf( 'Activating %s', previewingThemeName );
 			} else if ( disabled ) {
 				return __( 'Saved' );
 			} else if ( isDirty ) {
-				return __( 'Activate & Save' );
+				return sprintf( 'Activate %s & Save', previewingThemeName );
 			}
-			return __( 'Activate' );
+			return sprintf( 'Activate %s', previewingThemeName );
```

Comment:
With this PR, these strings are no longer translated. This needs to be fixed, @arthur791004.

Since the new strings involve interpolation, they require `// translators:` notes.

Reply:
Oops...

Reply:
Here is the fix: https://github.com/WordPress/gutenberg/pull/57147